src/sentry/static/sentry/app/components/forms/form.jsx

```diff
@@ -1,36 +1,106 @@
 import React from 'react';
+import underscore from 'underscore';
 
 import FormState from './state';
 import {t} from '../../locale';
 
-class Form extends React.Component {
+export default class Form extends React.Component {
+  static propTypes = {
+    onSubmit: React.PropTypes.func.isRequired,
+    onSubmitSuccess: React.PropTypes.func,
+    onSubmitError: React.PropTypes.func,
+    submitDisabled: React.PropTypes.bool,
+    submitLabel: React.PropTypes.string,
+    footerClass: React.PropTypes.string,
+    extraButton: React.PropTypes.element,
+    initialData: React.PropTypes.object,
+    fields: React.PropTypes.arrayOf(
```

Comment:
we could probably rm -rf this since its now unused, and im not sure i like the approach given context + fields